engines/scc_proxy/lib/scc_proxy/engine.rb

```diff
@@ -392,27 +399,22 @@ def deregister_hybrid(auth)
         def scc_deactivate_product
           auth = request.headers['HTTP_AUTHORIZATION']
           if @system.byos? && @product[:product_type] != 'base'
```

Comment:
What if system is byos, and the product is base, do we handle this in some other workflow?